### ISSUE TITLE
Start dogfooding iOS

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1475]
Everybody on the team should start dogfooding iOS and use Quiet as you go through your day. You can make a new temporary community if you want, until deletion is working.

**Comment by sbillig** [comment]
@holmesworcester Speaking of dogfood, is there an active community that y'all are willing to share?

**Comment by holmesworcester** [comment]
Yes! I'm hesitant to post a public link because we really do use it but don't have even the most basic moderation features yet. Though we'd like to and I hope to do it soon. (Ideally the app itself would invite you to join our community on the join page.)

Send me an email and I'll send you an invite! H@tryquiet.org
